packages/components/src/components/radio-button-wrapper/radio-button-wrapper.tsx

```diff
@@ -76,6 +77,14 @@ export class RadioButtonWrapper {
     });
   }
 
+  public componentShouldUpdate(
```

Comment:
Maybe change order of defined lifecycle hooks to reflect the real order. Here it is strange that `componentDidRender` is defined above but at runtime `componentShouldUpdate` happens before.